docs/guidelines.md

```diff
@@ -113,7 +113,30 @@ ngeo.exampleDirective = function(…) {
       // …
     },
     controllerAs: 'ctrl',
+    // …
+  });
+```
+
+But if you bind your directive to the controller and you declare your attribute
+as a two ways binding attrribute, you can use the `.` notation. See the example
```

Comment:
Yes. For instance, like here : https://github.com/camptocamp/ngeo/blob/master/src/directives/colorpicker.js (see colors attributes (both in the directive and in the controller) and the bindToController statement (in the directive).)